Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -413,6 +409,48 @@
     SodiumCarbonate: 4
 
 - type: reaction
+  id: Benzene
+  impact: Medium
+  reactants:
+    Hydrogen: 
+      Amount: 1
+    Carbon:
+      Amount: 1
+  products:
+    Benzene: 0.5
+
+- type: reaction
+  minTemp: 310 # Emo, you could change this to use one of your fancy machines, provided that machine is roundstart
+  id: Hydroxide
```

Comment:
I'm concerned that this recipe will conflict with too many other recipes that involve oxygen and hydrogen. You would need to test that all existing recipes are still makeable. Instead of having hydroxide as an intermediate, I think it would be better to not explicitly have hydroxide as a reagent but rather just require oxygen and hydrogen wherever hydroxide would be required.

Reply:
I tested all of them, and changed those that are needed.